gitflow-common

```diff
@@ -794,14 +794,21 @@ run_pre_hook() {
 #     post-flow-<subcmd>-<subaction>
 #
 # If such a hook script exists and is executable, it is called with the given
-# positional arguments.  Its return code is ignored.
+# positional arguments.  If its return code non-zero, the git-flow action is
+# aborted.
 #
 run_post_hook() {
-	local scriptfile
+	local scriptfile exitcode
 
 	scriptfile="${HOOKS_DIR}/post-flow-${SUBCOMMAND}-${SUBACTION}"
+	exitcode=0
```

Comment:
Why this one?
You check the `exitcode` only if `scriptfile` is executable, in which case three lines later you explicitly set `exitcode` to the return code of `scriptfile`....

Otherwise to me it looks OK.
:+1:

Reply:
@Shea690901 I'm following the exact pattern of `run_pre_hook`